amqp_connection.c

```diff
@@ -29,6 +29,7 @@
 #include "php.h"
 #include "php_ini.h"
 #include "ext/standard/info.h"
+#include "ext/standard/md5.h"
```

Comment:
I can’t think of an attack scenario but I would still like to not use MD5 anymore ever